src/unix/tcp.c

```diff
@@ -30,13 +30,15 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 
+#if !defined(_AIX)
```

Comment:
```suggestion
#if !defined(_AIX) && !defined(__PASE__)
```
It doesn't have ifaddrs.h either, or does it?